SpecFlowMarsProject/StepDefinitions/LanguagePageStepDefinitions.cs

```diff
@@ -0,0 +1,104 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using SpecFlowMarsProject.Pages;
+using SpecFlowMarsProject.Tests;
+using SpecFlowMarsProject.Utilities;
+using System;
+using System.Diagnostics;
+using TechTalk.SpecFlow;
+
+namespace SpecFlowMarsProject.StepDefinitions
+{
+    [Binding]
+    public class LanguagePageStepDefinitions : CommonDriver
+    {
+        //IWebDriver driver;
+        //IWebDriver driver = new ChromeDriver();
+        LoginPage loginPageObj = new LoginPage();
+        LanguagePage languageObj = new LanguagePage();
+
+        [BeforeScenario]
+        public void SetUpTimeMaterial()
+        {
+            //Open Chrome Browser
+            driver = new ChromeDriver();
+
+        }
+
+        [Given(@"I log into Mars portal and navigates to Homepage")]
+        public void GivenILogIntoMarsPortalAndNavigatesToHomepage()
+        {
+            loginPageObj.Loginpage(driver);
```

Comment:
Please do not pass the driver In action methods (methods which in pages doing things like create, delete etc ) instead get the pages to extend to common driver where you will create a common driver class which will initialise the web driver 

public class CommonDriver
    {
        public static IWebDriver driver;

        public void initialise()
        {
            // create driver
            driver = new ChromeDriver();

        }

    }